src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.c

```diff
@@ -21,49 +109,447 @@ const ActorInit Bg_Spot16_Bombstone_InitVars = {
     (ActorFunc)BgSpot16Bombstone_Update,
     (ActorFunc)BgSpot16Bombstone_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot16_Bombstone/func_808B4C30.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot16_Bombstone/func_808B4C4C.s")
+static InitChainEntry sInitChainBoulder[] = {
+    ICHAIN_F32(uncullZoneForward, 2000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+static InitChainEntry sInitChainDebris[] = {
+    ICHAIN_F32(gravity, 65535, ICHAIN_CONTINUE),          ICHAIN_F32(minVelocityY, 65526, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE), ICHAIN_F32(uncullZoneScale, 200, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+Vec3f velocity = { 0.0f, 0.0f, 0.0f };
+Vec3f acceleration = { 0.0f, 0.4f, 0.0f };
+
+f32 D_808B6074[] = { 66.0f, 51.0f, 48.0f, 36.0f, 21.0f };
+
+s16 D_808B6088[] = { 0, 1, 2, 3, 4 };
+
+void func_808B4C30(BgSpot16Bombstone* this) {
+    s16 params;
+
+    params = this->actor.params;
+    this->switchFlag = (params >> 8) & 0x3F;
+    this->actor.params = params & 0xFF;
+}
+
+void func_808B4C4C(BgSpot16Bombstone* this, GlobalContext* globalCtx) {
+    ColliderJntSph* colliderJntSph;
+
+    colliderJntSph = &this->colliderJntSph;
+    Collider_InitJntSph(globalCtx, colliderJntSph);
+    Collider_SetJntSph(globalCtx, colliderJntSph, &this->actor, &sJntSphInit, &this->colliderJntSphItem);
+    this->colliderJntSph.list->dim.worldSphere.center.x = this->actor.posRot.pos.x;
+    this->colliderJntSph.list->dim.worldSphere.center.y = this->actor.posRot.pos.y + 50.0f;
+    this->colliderJntSph.list->dim.worldSphere.center.z = this->actor.posRot.pos.z;
+    this->colliderJntSph.list->dim.worldSphere.radius = 0x78;
```

Comment:
```suggestion
    this->colliderJntSph.list->dim.worldSphere.radius = 120;
```
coordinates and distances make more sense in decimal imo